pkg/valueattribution/valueattribution.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package valueattribution
+
+import (
+	"context"
+	"time"
+)
+
+// PartnerInfo describing connector/partner key info in the database
+type PartnerInfo struct {
+	PartnerID  []byte
+	BucketName []byte
+	CreatedAt  time.Time
```

Comment:
I think we will just create this when inserting

Reply:
you mean pass these values as the insert()'s argument?